packages/app-project/src/screens/ClassifyPage/components/YourStats/YourStats.spec.js

```diff
@@ -1,15 +1,41 @@
-import { shallow } from 'enzyme'
+import { render, screen } from '@testing-library/react'
+import { composeStory } from '@storybook/react'
+import Meta, { YourStats } from './YourStats.stories.js'
+import { YourStatsStoreMock } from './YourStats.mock'
 
-import YourStats from './YourStats'
+describe('Component > YourStats', function () {
+  beforeEach(function () {
+		const YourStatsStory = composeStory(YourStats, Meta)
+		render(<YourStatsStory />)
+	})
 
-let wrapper
+	it('should have a heading', function () {
+    expect(screen.findByText(`Your ${YourStatsStoreMock.project.display_name} Statistics`)).to.be.ok()
+  })
 
-describe('Component > YourStats', function () {
-  before(function () {
-    wrapper = shallow(<YourStats />)
+	it('should have today\'s classification count ', function () {
+    expect(screen.getAllByText(`${YourStatsStoreMock.user.personalization.counts.today}`).length).to.equal(2)
+  })
+
+	it('should have this week\'s classification total ', function () {
+    expect(screen.queryByText(`${YourStatsStoreMock.user.personalization.counts.total}`)).to.be.ok()
+  })
+
+	it('should have a chart heading', function () {
+    expect(screen.findByText(`${YourStatsStoreMock.project.display_name} Daily Classification Count`)).to.be.ok()
```

Comment:
See Note 1 - I think findByText() should be replaced by getByText()

Also see Note 1a - I think the text we need to look for is "Classify.YourStats.DailyClassificationsChart.title"